### DISCUSSION TITLE
Can we customize css

### DISCUSSION THREAD

**Discussion by ujjwalguptaofficial** [thuliteio/doks#266] [Q&A] [answered]
Can we customize css e.g - primary color of button, menu color, dark theme or light theme color ? It would be great if i can customize it as per my product.

**Comment by h-enk** [comment, marked as answer]
Sure — set the Bootstrap variables in `./assets/scss/common/_variables.scss` and `./assets/scss/common/_dark.scss`.

**Reply by ujjwalguptaofficial** [reply]
works but only in starter theme as you have already mentioned lots of customization is not possible in child theme.

**Reply by h-enk** [reply]
Just copy them over and update as you wish. Child theme files override parent theme files.

**Reply by ujjwalguptaofficial** [reply]
@h-enk doesn't seem to work. First there is no file inside assets/common - so i copied & did some configuration in postcss.config.js but no luck. Could you please create an example or provide instructions .

**Reply by h-enk** [reply]
@ujjwalguptaofficial, see https://github.com/h-enk/doks-266 + https://doks-266.netlify.app/

**Reply by percula** [reply]
@h-enk any chance you can repost that repo? I just created a project using the child theme, copied the scss files as you mentioned, and nothing I do seems to affect the theme. I'm trying to change the primary color to green. Also, I'm wondering where the colors for the top accent bar are.

Thanks!

**Reply by AndrewBarba** [reply]
I'm having the same issue - can't seem to change theme colors with child theme

**Reply by h-enk** [reply]
Will get back to you — see also https://github.com/h-enk/doks/issues/748
